riscv-asm.md

```diff
@@ -190,12 +192,21 @@ Directive    | Arguments                      | Description
 .dtpreldword | expression [, expression]*     | 64-bit thread local word
 .sleb128     | expression                     | signed little endian base 128, DWARF
 .uleb128     | expression                     | unsigned little endian base 128, DWARF
-.p2align     | p2,[pad_val=0],max             | align to power of 2
-.balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
 .attribute   | name, value                    | RISC-V object attributes, more detailed description see [.attribute](#.attribute).
 
+## <a name=.align></a> `.align`
+
+The `.align` directive for RISC-V is an alias to `.p2align` which aligns to
```

Comment:
Perhaps this can be simplified to:

The interpretation of `.align` [depends on the architecture] .... The directive for RISC-V means ....

The verbose "This can be very confusing"  isn't necessary.